tutorials/scripting/cpp/build_system/scons.rst

```diff
@@ -73,3 +73,37 @@ There are two popular ways by which cross platform builds can be achieved:
 `godot-cpp-template <https://github.com/godotengine/godot-cpp-template>`__ contains an
 `example setup <https://github.com/godotengine/godot-cpp-template/tree/main/.github/workflows>`__
 for a GitHub based CI workflow.
+
+Using custom bindings
+---------------------
+
+Every branch of godot-cpp comes with bindings appropriate for the respective Godot version
+(e.g. the ``4.3`` branch comes with bindings compatible with Godot version ``4.3`` and
+later).
+
+However, you may want to use custom bindings, for example:
+
+* If you want to use the latest bindings from Godot ``master``.
+* If you want to use bindings exposed by custom modules.
```

Comment:
We should also include something allow the lines of "* If you build Godot yourself with different options than the official builds" because you should do this if you built with options like `scons precision=double` or `scons disable_3d=yes` or `scons deprecated=no` as well

Reply:
Good point!